akka-javasdk/src/main/scala/akka/javasdk/impl/SdkRunner.scala

```diff
@@ -617,7 +619,7 @@ private final class Sdk(
 
       override val serviceInfo: SpiServiceInfo =
         new SpiServiceInfo(
-          serviceName = sdkSettings.devModeSettings.map(_.serviceName).getOrElse(""),
+          serviceName = serviceNameOverride.orElse(sdkSettings.devModeSettings.map(_.serviceName)).getOrElse(""),
```

Comment:
... and this changed in https://github.com/akka/akka-sdk/commit/898ec6f98d8fb7ed8709cf5e88b2e35db9da1613#diff-26d7c802dc1ff7174ab9cbed72a852b5d31ce32af07c96b7686aa89a67fc328cR447

@efgpinto pls confirm that this is correct